src/main/java/org/folio/util/TokenCookieParser.java

```diff
@@ -0,0 +1,58 @@
+package org.folio.util;
+
+import org.apache.commons.lang3.StringUtils;
+import org.folio.rest.impl.LoginAPI;
+
+public class TokenCookieParser {
```

Comment:
Is there no standard library for parsing cookie headers?

Reply:
One would think there is. I didn't find one and is making the "same" work in Okapi. https://github.com/folio-org/okapi/pull/1251

Reply:
Added netty's `ServerCookieDecoder`.